cogs/cex.py

```diff
@@ -0,0 +1,195 @@
+import asyncio
+import json
+import aiohttp
+import discord
+from discord.ext import commands
+import time
+import urllib.parse
+import re
+
+cexRed = 0xff0000
+cexLogo = 'https://uk.webuy.com/_nuxt/74714aa39f40304c8fac8e7520cc0a35.png'
+cexEmoji = '<:cex:292749632025657354>'
+# Search item aliases
+D = {
+    'bI'    :   'boxId',
+    'bN'    :   'boxName',
+    'iMB'   :   'isMasterBox',
+    'cI'    :   'categoryId',
+    'cN'    :   'categoryName',
+    'cFN'   :   'categoryFriendlyName',
+    'sCI'   :   'superCatId',
+    'sCN'   :   'superCatName',
+    'sCFN'  :   'superCatFriendlyName',
+    'cB'    :   'cannotBuy',
+    'iNB'   :   'isNewBox',
+    'sP'    :   'sellPrice',
+    'cP'    :   'cashPrice',
+    'eP'    :   'exchangePrice',
+    'bR'    :   'boxRating',
+    'oOS'   :   'outOfStock',
+    'oOES'  :   'outOfEcomStock',
+    'eQOH'  :   'ecomQuantityOnHand'
+}
+
+class Cex(commands.Cog):
+
+    def __init__(self, client):
+        self.client = client
+
+    # Events
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print('Cex search cog online')
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        # we do not want the bot to reply to itself or other bots
+        if message.author.bot == True or message.author.id == self.client.user.id:
+            return
+
+        if 'https://uk.webuy.com/' in message.content:
+            product_id = message.content.split("id=",1)[1].split("&",1)[0].split(' ')[0]

```

Comment:
don't do this hacky shit, use https://docs.python.org/3/library/urllib.parse.html